test/cypress/integration/simplified-account/service-settings/stripe-details/task-summary.cy.js

```diff
@@ -0,0 +1,277 @@
+const userStubs = require('../../../../stubs/user-stubs')
+const gatewayAccountStubs = require('../../../../stubs/gateway-account-stubs')
+const stripeAccountSetupStubs = require('../../../../stubs/stripe-account-setup-stub')
+const { checkTaskNavigation, checkDisplayedTasks, taskStatus } = require('./task-summary-test-helpers')
+const { SANDBOX, STRIPE } = require('../../../../../../app/models/payment-providers')
+
+const USER_EXTERNAL_ID = 'user-123-abc'
+const SERVICE_EXTERNAL_ID = 'service-456-def'
+const SERVICE_NAME = {
+  en: 'My Cool Service', cy: 'Fy Ngwasanaeth Cwl'
+}
+const LIVE_ACCOUNT_TYPE = 'live'
+const GATEWAY_ACCOUNT_ID = 10
+
+const SERVICE_SETTINGS_URL = `/simplified/service/${SERVICE_EXTERNAL_ID}/account/${LIVE_ACCOUNT_TYPE}/settings`
+
+const setStubs = (opts = {}, additionalStubs = []) => {
+  cy.task('setupStubs', [
+    userStubs.getUserSuccess({
+      userExternalId: USER_EXTERNAL_ID,
+      gatewayAccountId: GATEWAY_ACCOUNT_ID,
+      serviceName: SERVICE_NAME,
+      serviceExternalId: SERVICE_EXTERNAL_ID,
+      role: opts.role,
+      features: 'degatewayaccountification' // TODO remove features once simplified accounts are live
+    }),
+    gatewayAccountStubs.getAccountByServiceIdAndAccountType(SERVICE_EXTERNAL_ID, LIVE_ACCOUNT_TYPE, {
+      gateway_account_id: GATEWAY_ACCOUNT_ID, type: LIVE_ACCOUNT_TYPE, payment_provider: opts.paymentProvider || STRIPE
+    }),
+    ...additionalStubs])
+}
+
+describe('Stripe details settings', () => {
+  beforeEach(() => {
+    cy.setEncryptedCookies(USER_EXTERNAL_ID)
+  })
+  describe('The task summary page', () => {
+    describe('For an admin user', () => {
+      beforeEach(() => {
+        setStubs({}, [
+          stripeAccountSetupStubs.getServiceAndAccountTypeStripeSetupSuccess({
+            serviceExternalId: SERVICE_EXTERNAL_ID,
+            accountType: LIVE_ACCOUNT_TYPE,
+            bankAccount: true
+          })
+        ])
+        cy.visit(SERVICE_SETTINGS_URL + '/stripe-details')
+      })
+      it('should show the correct title', () => {
+        cy.title().should('eq', 'Settings - Stripe details - GOV.UK Pay')
+      })
+      it('should show the correct heading', () => {
+        cy.get('h1').should('contain', 'Stripe details')
+      })
+      describe('The settings navigation', () => {
+        it('should show stripe details', () => {
+          cy.get('.service-settings-nav')
+            .find('li')
+            .contains('Stripe details')
+            .then(li => {
+              cy.wrap(li)
+                .should('have.attr', 'href', `${SERVICE_SETTINGS_URL}/stripe-details`)
```

Comment:
What part of the UI is this testing?

Reply:
It's testing that the stripe details link is visible in the service settings navigation